data-prepper-plugins/saas-source-plugins/saas-crawler/src/test/java/org/opensearch/dataprepper/plugins/source/saas_crawler/coordination/state/SaasWorkerProgressStateTest.java

```diff
@@ -0,0 +1,34 @@
+package org.opensearch.dataprepper.plugins.source.saas_crawler.coordination.state;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.Test;
+
+public class SaasWorkerProgressStateTest {
+
+    private static final ObjectMapper objectMapper = new ObjectMapper();
+
+    @Test
+    void testDefaultValues() throws JsonProcessingException {
+        String state = "{}";
+        SaasWorkerProgressState workderProgressState = objectMapper.readValue(state, SaasWorkerProgressState.class);
+        assert workderProgressState.getTotalItems() == 0;
```

Comment:
Please replace all `assert` calls with Hamcrest `assertThat();`

Reply:
switched to using hamcrest